app/models/concerns/workflow_pipelineable.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# Include in records to associate them with a pipeline via an enum
+module WorkflowPipelineable
+  extend ActiveSupport::Concern
```

Comment:
I think it is better to use pipelineable? Not sure of the changes we need to make.

Reply:
Thanks for the review. 
I was considering the WorkflowPipelineable concern, as the pipelines added in workflows differ from the existing Pipelinable concern, which is used in many other places. 
Thought it would be better to keep the two concerns separate as the workflow enum pipeline: { pacbio: 0, ont: 1, extraction: 2, sample_qc: 3, hic: 4, bio_nano: 5 }. has only two entries in common with Pipelinable—pacbio and ont. 
Also, I was thinking whether changing this enum would cause issues with Pipelinable, since it is used in many other places.